src/print-prescription/print-prescription.scss

```
@@ -0,0 +1,48 @@
+@use '@carbon/layout';
+@use '@carbon/type';
+@use '@carbon/colors';
+
+.title {
+  @include type.type-style('heading-02');
+}
+
+.btnSet {
+  width: 100%;
+}
+
+.faintText {
+  color: colors.$gray-50;
+  @include type.type-style('heading-01');
+}
+
+.printOutContainer {
+  background-color: colors.$gray-20;
+  padding: layout.$spacing-05;
+  width: 320px;
```

```suggestion
  width: 320px;
```

nit: `20rem` is easier to reason about here.